src/slices/AppSlice.js

```diff
@@ -43,7 +43,20 @@ export const loadAppDetails = createAsyncThunk("app/loadAppDetails", async ({ ne
   }
 
   const stakingTVL = parseFloat(graphData.data.protocolMetrics[0].totalValueLocked);
-  const marketPrice = parseFloat(graphData.data.protocolMetrics[0].ohmPrice);
+  // NOTE (appleseed): marketPrice from Graph was delayed, so get CoinGecko price
+  // const marketPrice = parseFloat(graphData.data.protocolMetrics[0].ohmPrice);
+  let marketPrice;
+  try {
+    const originalPromiseResult = await dispatch(
+      loadMarketPrice({ networkID: networkID, provider: provider }),
+    ).unwrap();
```

Comment:
unclear whats happening around here.

Reply:
loadMarketPrice is an asyncThunk so that we can update it separately in the app slice